src/MSBuild/XMake.cs

```diff
@@ -1752,11 +1752,13 @@ private static List<BuildManager.DeferredBuildMessage> GetMessagesToLogInBuildLo
             NativeMethodsShared.LongPathsStatus longPaths = NativeMethodsShared.IsLongPathsEnabled();
             if (longPaths != NativeMethodsShared.LongPathsStatus.NotApplicable)
             {
+                string resourceString = "LongPaths_" + longPaths.ToString();
                 messages.Add(
                     new BuildManager.DeferredBuildMessage(
                         ResourceUtilities.FormatResourceStringIgnoreCodeAndKeyword(
-                        "LongPathsEnabled",
-                        longPaths == NativeMethodsShared.LongPathsStatus.Enabled),
+                            "LongPaths",
+                            ResourceUtilities.FormatResourceStringIgnoreCodeAndKeyword(
+                                "LongPaths_" + longPaths.ToString())),
```

Comment:
Accidental repetition

Reply:
It's in fact what I wanted. but I see it's confusing so I'll remove the nested resource string.